src/components/BSelect/BSelect.vue

```diff
@@ -61,25 +61,43 @@ export default {
   name: 'b-select',
   validator: null,
   props: {
+    /**
+    * Component Id
+    */
     id: {
       type: String,
       required: true
     },
+    /**
+    * Array of objects to be displayed as options.
+    */
     selectOptions: {
       type: Array
     },
+    /**
+    * Use to turn on validation and to indicate field is required.
+    */
     required: {
       type: Boolean,
       required: true
     },
+    /**
+    * Label of component.
+    */
     selectLabel: {
       type: String,
       required: true
     },
+    /**
+    * If included, this object will be pre-selected.
```

Comment:
It should be clear here that this needs to be the id of one of the selectOptions

Reply:
I'll add this in my refactor branch

Reply:
its actually the entire object.